test/sanity/code-smell/shebang.py

```diff
@@ -30,6 +30,10 @@ def main():
     }
 
     skip = set([
+        'test/integration/targets/collections/collection_root_user/ansible_collections/testns/testcoll/plugins/modules/win_csbasic_only.ps1',
```

Comment:
Are these getting picked up because they are not in a `library` dir?

Reply:
Yep; collections calls that dir `modules`